api/wrappers.py

```diff
@@ -0,0 +1,34 @@
+from functools import wraps
+
+import jwt
+from flask import request, jsonify, current_app
+
+from db import User
+
+
+def token_required(function):
+    @wraps(function)
+    def decorated(*args, **kwargs):
+        token = None
+
+        if 'token' in request.headers:
+            token = request.headers['token']
+
+        if not token:
+            return jsonify({'message': 'Token is missing!'})
+
+        try:
+            data = jwt.decode(
+                token,
+                current_app.config['SECRET_KEY'],
+                algorithms=['HS256']
+            )
+            current_user = User.query.filter_by(id=data['id']).first()
+        except jwt.ExpiredSignatureError:
+            return jsonify({'message': 'Your token expired! \
```

Comment:
As you can see, `jsonify({'message': ..` is everywhere, let's do something with it. At least move that to some function e.g jsonify_message in utils like [here](https://github.com/CiscoSecurity/tr-05-serverless-google-safe-browsing/blob/b656485707089190e1c805b92ac22b60b57659ea/code/api/utils.py#L138)

Also, you can create custom exception with the same logic and do some app level handler for it. Please take a look [here](https://github.com/CiscoSecurity/tr-05-serverless-farsight-dnsdb/blob/04b14a46bc05d108ab6f43a43b3e774c563aeb98/api/errors.py ) and [here](https://github.com/CiscoSecurity/tr-05-serverless-farsight-dnsdb/blob/04b14a46bc05d108ab6f43a43b3e774c563aeb98/app.py#L36)

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/0217f91d71366d0c783fcf70ff61e1512d8ef806